website/docs/cloud-docs/workspaces/settings/vcs.mdx

```diff
@@ -4,81 +4,94 @@ page_title: VCS Connections - Workspaces - Terraform Cloud and Terraform Enterpr
+Terraform Cloud uses your VCS provider's API to retrieve the changed files in your repository. You can choose one of the following options to specify which changes trigger Terraform runs.
 
-For workspaces that specify a Terraform working directory, Terraform Cloud assumes that only some content in the repository is relevant to the workspace. Only changes that affect the relevant content will trigger a run. This behavior also applies to [speculative plans](/cloud-docs/run/#speculative-plans) on pull requests — Terraform Cloud won't queue plans for changes that aren't considered relevant.
+#### Always trigger runs
 
-By default, only the designated working directory is considered relevant.
+This option instructs Terraform to begin a run when changes are pushed to any file within the repository. This can be useful for repositories that do not have multiple configurations but require a working directory for some other reason. However, we do not recommend this approach for true monorepos, as it queues unnecessary runs and slows down your ability to provision infrastructure.
 
-You can adjust this behavior in two ways:
+#### Only trigger runs when files in specified paths change
 
-- **Add more trigger directories.** Terraform Cloud will queue runs for changes in any of the specified trigger directories (including the working directory).
+This option instructs Terraform to begin new runs only for changes that affect specified files and directories. This behavior also applies to [speculative plans](/cloud-docs/run/#speculative-plans) on pull requests.
 
-  For example, if you use a top-level `modules` directory to share Terraform code across multiple configurations, changes to the shared modules are relevant to every workspace that uses that repo. You can add `modules` as a trigger directory for each workspace to make sure they notice any changes to shared code.
-- **Mark the entire repository as relevant.** If you set the "Automatic Run Triggering" setting to "Always Trigger Runs," Terraform Cloud will assume that anything in the repository might affect the workspace's configuration, and will queue runs for any change.
+You can use trigger patterns and trigger prefixes in the **Add path** field to specify groups of files and directories.
+- **Trigger Patterns:** (Recommended) Use glob patterns to specify the files that should trigger a new run. For example, `/submodule/**/*.tf`, specifies all files with the `.tf` extension that are nested below the `submodule` directory. You can also use more complex patterns like `/**/networking/**/*`, which specifies all files in the `networking` folder at this level of the file path (e.g., `/submodule/service-1/networking/private/main.tf`). Refer to [Glob Patterns for Automatic Run Triggering](#glob-patterns-for-automatic-run-triggering) for details.
```

Comment:
```
which specifies all files in the `networking` folder at this level of the file path
```
This sentence is a bit incorrect and I would revert it back to read something like:
```
which specifies all files which have a `networking` folder on their file path
```

@laurapacilio what do you think of it?

Reply:
1c7e5b6d6bdf7616b3f82a3846f96dbefe734d45